packages/html-handler/src/utils.ts

```diff
@@ -1,13 +1,25 @@
 import * as crypto from 'crypto';
-import { LogEntry, LogEntryQueue } from './types';
+import { v4 } from 'node-uuid';
 
-export function findDefiningEntry(queue: LogEntryQueue): LogEntry {
+import { LogEntry, TContextBase } from '@debugr/core';
+import { LogEntryQueue } from './types';
+
+export function findDefiningEntry<
+  TContext extends TContextBase = { processId: string },
+  TGlobalContext extends Record<string, any> = {},
+>(
+  queue: LogEntryQueue<Partial<TContext>, TGlobalContext>,
+): LogEntry<Partial<TContext>, TGlobalContext> {
   if (queue.firstOverThreshold !== undefined) {
     return queue.entries[queue.firstOverThreshold];
   } else if (!queue.entries.length) {
     return {
-      ts: 0,
+      ts: new Date(),
```

Comment:
that is not the same!! dej `new Date(0)` když už

Reply:
(ačkoliv.. asi je to fuk)

Reply:
done